Cargo.toml

```diff
@@ -59,6 +60,14 @@ path = "sdk/core/azure_core_macros"
 version = "0.6.0"
 path = "sdk/core/azure_core_amqp"
 
+[workspace.dependencies.azure_messaging_eventhubs]
+version = "0.6.0"
+path = "sdk/eventhubs/azure_messaging_eventhubs"
+
+[workspace.dependencies.azure_messaging_eventhubs_checkpointstore_blob]
+version = "0.1.0"
+path = "sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob"
+
```

Comment:
Is this a dependency for anything? We should only do this for dependencies because they can be a bigger pain to update.

Reply:
No it is not a dependency for something else.